src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiEnchantColour.java

```diff
@@ -422,6 +422,9 @@ private void renderSideBar(int mouseX, int mouseY, float partialTicks) {
 
 	@Override
 	protected void keyTyped(char typedChar, int keyCode) throws IOException {
+		if (Keyboard.isKeyDown(Keyboard.KEY_ESCAPE)) {
```

Comment:
Using onGuiClose would probably be better

Reply:
Ha, first